### PR TITLE
New constant for federated protocols

### PR DESCRIPTION
**Pull request by annando**
A new constant is added to the protocols that covers all federated protocols.

### CODE REVIEW

File: src/Model/Contact.php [hunk 1]
```diff
@@ -1587,7 +1587,7 @@ public static function getPostsFromUrl($contact_url, $thread_mode = false, $upda
 			return '';
 		}
 
-		if (in_array($contact["network"], [Protocol::ACTIVITYPUB, Protocol::DFRN, Protocol::DIASPORA, Protocol::OSTATUS, ""])) {
+		if (in_array($contact["network"], array_merge(Protocol::FEDERATED ,['']))) {
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Why the empty element?

File: src/Model/Contact.php [hunk 2]
```diff
@@ -1878,7 +1877,8 @@ public static function updateFromProbe($id, $network = '', $force = false)
 		self::updateContact($id, $uid, $ret['url'], $ret);
 
 		// Update the corresponding gcontact entry
-		PortableContact::lastUpdated($ret["url"]);
+//		PortableContact::lastUpdated($ret["url"]);
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Why the comment?

Comment (annando) [reply src/Model/Contact.php]: Because I forgot to remove that line :grin:

File: src/Model/Contact.php (hunk 1 above)
Comment (annando) [reply src/Model/Contact.php]: I'm currently unsure. But I wanted to make this PR as non breaking as possible, so I kept it.

File: src/Model/Contact.php (hunk 2 above)
Comment (annando) [reply src/Model/Contact.php]: In fact I'm currently working at some GContact stuff and got drifted away. I think you know that problem :grimacing: 

So I hold back my changes in the gcontact file and wanted to commit this thing here first.

File: src/Model/Contact.php (hunk 1 above)
Comment (MrPetovan) [reply src/Model/Contact.php]:
```suggestion
		if (empty($contact["network"]) || in_array($contact["network"], Protocol::FEDERATED))) {
```

**Pull request merged**